client/src/app/(modules)/network/(form)/new/organisation/input-component.tsx

```diff
@@ -68,14 +74,14 @@ const InputComponent = ({
     );
   }
   if (type === 'textarea') {
-    const watchField = watch(field.name);
-    const counterId = `${field.name}-counter`;
+    const watchField = watch(name) as string;
+    const counterId = `${name} - counter`;
```

Comment:
You're using this variable on line 94 as a HTML id as well so having spaces will give you some issues.